src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -31,11 +32,10 @@
    (contains? event "old_value")
    (contains? event "resource_type")))
 
-(defn report?
+(pls/defn-validated report? :- report/report-schema
```

Comment:
I don't think we should use the other functions in this file for testing as good examples of how to do this, they are pre-prismatic schema also. And yeah, this isn't a test any more - its keywordizes as well ... which is kind of redundant right?